client/util.go

```diff
@@ -43,17 +45,17 @@ func CreateEthRandomKeysBatch(n int) []*ethereum.SignKeys {
 }
 
 type keysBatch struct {
-	Keys      []signKey `json:"keys"`
-	CensusID  string    `json:"censusId"`
-	CensusURI string    `json:"censusUri"`
+	Keys      []signKey        `json:"keys"`
+	CensusID  tmbytes.HexBytes `json:"censusId"`
```

Comment:
Is this a reasonable trade-off? I mean, we are not depending on a tendermint library for a package that has nothing to do with tendermint. Shouldn't make more sense to have our "HexBytes" type instead?